app/controllers/notes_controller.rb

```diff
@@ -7,13 +7,16 @@ class NotesController < NestedNodeResourceController
   include MultipleDestroy
   include NodesSidebar
   include NotificationsReader
+  include Previewable
 
-  before_action :find_or_initialize_note, except: [:index, :new, :multiple_destroy]
+  before_action :find_or_initialize_note, except: [:index, :new, :multiple_destroy, :preview]
   before_action :initialize_nodes_sidebar, only: [:edit, :new, :show]
   before_action :set_auto_save_key, only: [:new, :create, :edit, :update]
+  before_action :set_form_preview_path, only: [:create, :edit, :update]
 
   def new
     @note = @node.notes.new
+    set_form_preview_path
```

Comment:
Why are we manually calling it here instead of the before_action callback and can we make it so the callback works here?

Reply:
Because we're setting `@note` inline instead of in the before action, but I don't see any reason for that so I've updated the new action to use the callbacks instead